packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -11,6 +11,9 @@ import { assert, details } from '@agoric/assert';
 import { isPromise } from '@agoric/promise-kit';
 import { insistVatType, makeVatSlot, parseVatSlot } from '../parseVatSlots';
 import { insistCapData } from '../capdata';
+import { makeVirtualObjectManager } from './virtualObjectManager';
+
+const VIRTUAL_OBJECT_CACHE_SIZE = 3; // XXX ridiculously small value to force churn for testing
```

Comment:
Probably.  I haven't yet taken the time to work out the right way to inject it, but since it's set on a per-vat basis I was figuring it should be a creation parameter that can be specified in the config file.